source/includes/_accounts.md

```diff
@@ -149,7 +149,7 @@ print(response.text)
 }
 ```
 
-`PUT https://api.apollo.io/v1/accounts`
+`PUT https://api.apollo.io/v1/accounts/YOUR_ACCOUNT_ID`
 
```

Comment:
Do, We need to remove 'id' from parameters table below this?

Reply:
Ya sure, removing that as well.

Reply:
Done, please check. @PavanSankavar